app/views/ems_cloud/arbitration_profile_edit.html.haml

```diff
@@ -0,0 +1,7 @@
+= render :partial => "layouts/flash_msg"
+%div{'ng-controller' => 'arbitrationProfileFormController', :id => 'angular_form_div'}
```

Comment:
this could be `#angular_form_div{'ng-controller' => 'arbitrationProfileFormController'}` instead..